src/textual/_ansi_sequences.py

```diff
@@ -299,3 +304,16 @@
     "\x1b[1;8x": (Keys.Escape, Keys.ControlShift8),
     "\x1b[1;8y": (Keys.Escape, Keys.ControlShift9),
 }
+
+
+# Mapping of escape codes to report whether they support a "mode" we requested.
+ANSI_SEQUENCES_MODE_REPORTS: Dict[str, Tuple[Mode, ModeReportParameter]] = {
```

Comment:
Are we capturing all possible (now and future) responses? I have a feeling that a single regex would be more future safe option rather than enumerating all possible combinations.